modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,138 @@
+* Secondary networks: Supported as a post-installation task only. A third-party  plugin servers as the CNI and physical interfaces connect to a customized `br-ex` bridge by use of a vLAN. You define the SLB configuration in a `NodeNetworkConfigurationPolicy` (NNCP) file.
+* Localnet topology networks: Supported as a post-installation task only. A localnet topology froms part of a secondary network. A localnet topology allows connections to the physical network, but requires additional configuration of the underlying OVS bridge on cluster nodes. You define the SLB configuration in either a `NetworkAttachmentDefinition` (NAD) or a `NodeNetworkConfigurationPolicy` file.
+
+You can use either kernel bonding or OVS bonding to enable the `balance-slb` mode on your chosen network type. The following list details the differences between the kernel bonding and the OVS bonding methods:
+
+* Kernel bonding: Useful for third-party CNI environments but requires additional network configurations, such as condiguring an additional Linux bridge, to support `balance-slb` mode. You can use this method external to your {product-title} cluster, such as communicating to a {op-system-base-full} server. Performance issues notable when compared to the OVS bonding method.
+* OVS bonding: Useful for the OVN-Kubernetes CNI plugin. Natively supports `balance-slb` mode and integrates . All node traffic diverts to a single `balance-slb` mode interface. You cannot use the method external to your {product-title} cluster. This method performs better than the Kernel bonding method.
+
+[IMPORTANT]
+====
+If you use the kernel bonding method to enable `balance-slb` mode for your cluster, Network Manager might experience Gratuitous Address Resolution Protocol (GARP) handling issues. These issues occur when the Network Manager cannot retrieve source MAC addresses from the OVS bridge interface because of a broken link to a port disables the port.
+====
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console.
+* You have installed the {oc-first}.
+* You have logged into {product-title} as a cluster administrator.
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in the `MachineConfig` file.
+* You created a manifest object and defined a customized `br-ex` bridge in the object configuration file.
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in the NNCP or NAD file.
+* You created a NNCP CR and defined a customized `br-ex` bridge network configuration in the object configuration file.
+
+.Procedure
+
+. Choose one of the following methods:
++
+.. For OVS bonding, open your NMState configuration file in your editor and add the following configuration in the file:
```

Comment:
Before they do this, they need to also create a baremetal-specific day one network config in install-config (or a secret if it's day 2). That's the tricky part of this process - you need one config to pull ignition, then you need another config in that ignition that does the rest of the balance-slb configuration.

I'll send you an example of the initial config. It's much simpler than what's below because it only needs to bring up one of the interfaces and doesn't need to configure any OVS objects.